frontend/src/components/Layout/Header/ConfirmLogOut/styled.tsx

```diff
@@ -0,0 +1,64 @@
+import styled from 'styled-components';
+import { Button } from 'components/Button';
+import { DialogTitle } from '@mui/material';
+
+import theme from 'themeSettings';
+
+export const ConfirmBox = styled('div')({
+  minWidth: '660px',
+  minHeight: '335px',
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'start',
+  margin: '0 !important',
+  padding: '0 !important',
+});
+
+export const ConfirmMessage = styled(DialogTitle)({
+  display: 'inline-block',
```

Comment:
Lets save this attr - we have universal attr-prop, which can include as inline style, and some of block style. Maybe it will useful for styling, scaling, positions etc in future.